### PR TITLE
Planning: updates

### PR DESCRIPTION
**Pull request by panjiacheng**

### CODE REVIEW

File: modules/planning/tasks/deciders/st_bounds_decider/st_obstacles_processor.cc [hunk 1]
```diff
@@ -64,20 +69,51 @@ Status STObstaclesProcessor::MapObstaclesToSTBoundaries(
...
+  std::tuple<std::string, STBoundary, Obstacle*> closest_stop_obstacle;
+  std::get<0>(closest_stop_obstacle) = "NULL";
   for (const auto* obs_item_ptr : path_decision->obstacles().Items()) {
     Obstacle* obs_ptr = path_decision->Find(obs_item_ptr->Id());
     CHECK_NOTNULL(obs_ptr);
 
     std::vector<STPoint> lower_points;
     std::vector<STPoint> upper_points;
     if (!ComputeObstacleSTBoundary(*obs_ptr, &lower_points, &upper_points)) {
+      // Obstalce doesn't appear on ST-Graph.
```
Comment (xiaoxq) [inline modules/planning/tasks/deciders/st_bounds_decider/st_obstacles_processor.cc]: typo

File: modules/planning/tasks/deciders/st_bounds_decider/st_obstacles_processor.cc [hunk 2]
```diff
@@ -64,20 +69,51 @@ Status STObstaclesProcessor::MapObstaclesToSTBoundaries(
...
-    obs_ptr->set_path_st_boundary(boundary);
+    if (obs_ptr->Trajectory().trajectory_point_size() == 0) {
+      // Obstacle is static.
+      if (std::get<0>(closest_stop_obstacle) == "NULL" ||
+          std::get<1>(closest_stop_obstacle).bottom_left_point().s() >
+              boundary.bottom_left_point().s()) {
+        // If this static obstacle is closer for ADC to stop, record it.
+        std::get<0>(closest_stop_obstacle) = obs_ptr->Id();
+        std::get<1>(closest_stop_obstacle) = boundary;
+        std::get<2>(closest_stop_obstacle) = obs_ptr;
```
Comment (xiaoxq) [inline modules/planning/tasks/deciders/st_bounds_decider/st_obstacles_processor.cc]: Try `make_tuple` to assign easily.
https://en.cppreference.com/w/cpp/utility/tuple/make_tuple

Comment (xiaoxq) [reply modules/planning/tasks/deciders/st_bounds_decider/st_obstacles_processor.cc]: And also other stuffs like `std::tie` which would be helpful when you work with tuple.
https://en.cppreference.com/w/cpp/utility/tuple/tie

File: modules/planning/tasks/deciders/st_bounds_decider/st_obstacles_processor.cc (hunk 1 above)
Comment (panjiacheng) [reply modules/planning/tasks/deciders/st_bounds_decider/st_obstacles_processor.cc]: fixed

File: modules/planning/tasks/deciders/st_bounds_decider/st_obstacles_processor.cc (hunk 2 above)
Comment (panjiacheng) [reply modules/planning/tasks/deciders/st_bounds_decider/st_obstacles_processor.cc]: fixed. thx for this.

**Pull request merged**